frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1478,13 +1479,12 @@ const ManageHostsPage = ({
       return <TableDataError />;
     }
 
-    // Hosts have not been set up for this instance yet.
+    // There are no hosts for this instance yet
     if (
-      (getStatusSelected() === ALL_HOSTS_LABEL && selectedLabel.count === 0) ||
```

Comment:
This appears to be the offending line of code. I didn't see a reason it couldn't be removed and just rely on the condition second. I removed it and manual QA didn't show any issues, but I wanted to check with you to make sure it's not an edge case I'm missing.